Steepfile

```diff
@@ -1,8 +1,48 @@
+# Declare shortcuts for Steep::Signature::Ruby to make this file easier to read
+# as well as facilitating the findability of violation types emitted by the CLI
+# (e.g. the CLI emits `Diagnostic ID: Ruby::UnknownConstant` when finding errors).
+Ruby = Steep::Diagnostic::Ruby
```

Comment:
Not a deal-breaker, but `Diagnostic` might suite better here.

Examples

```ruby
  configure_code_diagnostics(Diagnostic.strict) do |config|
    # ...
    config[Diagnostic::FallbackAny] = :information
  end
```

Reply:
The problem with this one is that it is not `Diagnostic::FallbackAny`, it would be `Diagnostic::Ruby::FallbackAny`. It gets repetitive.

Tbh, this is what Steep does themselves: https://github.com/soutaro/steep/blob/81da4a1172fa61551ec4f0d871fb5ed7cd588d2f/Steepfile#L1

Reply:
I'm ok updating it as a follow up if we feel strongly about it.